DemocracyClub/Website
# Merge django-hermes into website repo

> Django-hermes is a fork of a mostly unmaintained Django package. We’ve taken the fork in a different direction to the upstream project, and we only use it on the DC website. This means it’s actually just annoying to have it as a different project. 
> 
Here, we've removed the external `django-hermes` package, consolidated requirements and updated imports, as needed. 
> 
> To test:
> In a new virtual env / new project set up to make sure that the old package isn’t installed, and isn’t needed.
> Testing an existing database with blog posts to make sure the posts still work.
> Testing on stage too

https://stage.democracyclub.org.uk/blog
